src/Ops.Web/Areas/Admin/Views/Locations/_AddGroupsPartial.cshtml

```diff
@@ -0,0 +1,47 @@
+﻿@model Ocuda.Ops.Controllers.Areas.Admin.ViewModels.Location.GroupListViewModel;
+
+@if (Model.Groups.Count() == 0)
+{
+    <div class="alert alert-warning">No Items found</div>
+}
+else
+{
+    <div>
+        <table class="table table-sm table-bordered table-striped">
+            <thead>
+                <tr>
+                    <th>Group Type</th>
+                    <th width="100px">Is Location Region</th>
```

Comment:
See above about widths. Should avoid sizing at all if possible, avoid sizing in pixels, and avoid sizing without classes.